kie-server-parent/kie-server-tests/kie-server-integ-tests-controller/src/test/java/org/kie/server/integrationtests/controller/KieControllerRuleCapabilitiesIntegrationTest.java

```diff
@@ -110,6 +117,8 @@ public void testStartScannerNotExistingContainer() {
             fail("Should throw exception about container not found.");
         } catch (KieServerControllerClientException e) {
             assertNotFoundException((T) e);
+            assertEquals("No container spec found for id not-existing",
```

Comment:
In this test class is already imported org.assertj.core.api.Assertions.assertThat so there is no need to import JUnit asserts. Please replace assertEquals(epxpected, actual) with assertThat(actual).isEqualTo(Expected).

Reply:
Done.